docs/self-hosted/kubernetes-embedded.md

```diff
@@ -24,14 +24,19 @@ in an existing Kubernetes cluster using Helm follow [this guide](./kubernetes-he
     * Ports 80, 443, and 8800 must be accessible, for a full list of ports
     * Must be reachable at the chosen domain names from your local machine
 1. If your firewall restricts external connections the following domains must be accessible from the server:
-  * hub.docker.com
-  * proxy.replicated.com
+  * index.docker.io
```

Comment:
Is this still needed with the *.docker.io below?

Reply:
Replicated lists them separate. My understanding is that it isn't but was just trying to follow this as closely as possible
<img width="308" alt="Screenshot 2024-11-22 at 17 05 53" src="https://github.com/user-attachments/assets/0812b325-0c5c-4fee-88e1-a39408ac79a3">

Reply:
Yeah, just noticed that too in the docs. Then let's keep it like that